docs/source/install/docker-compose.rst

```diff
@@ -0,0 +1,275 @@
+    # If the Registry Harvest CLI is executed with the option to download test data, then this directory will be
+    # cleaned-up and populated with test data.
+    # Also, this `HARVEST_DATA_DIR` location should be accessible from the docker containers of the Registry Harvest Service
+    # and Registry Crawler Service.
+    HARVEST_DATA_DIR=./test-data/registry-harvest-data
+
+.. note::
+    The `HARVEST_DATA_DIR` specified above should have the directories of the data to be harvested.
+
+    An example of a `harvest-job-config.xml` file (to set as `HARVEST_JOB_CONFIG_FILE` in the above configurations)
+    can be obtained from the `<REGISTRY_ROOT>/docker/default-config` directory.
+
+5. Generate the self-signed certificates as follows.
+
+
+.. note::
+    Certificates are required for OpenSearch to enable https.
+..
+
+    a) Change the current working directory to `<REGISTRY_ROOT>/docker/certs` directory. ::
+
+        cd certs
+
+    b) Generate self-signed certificates by executing the following script. ::
+
+        ./generate-certs.sh
+
+    c) Change the current working directory to `<REGISTRY_ROOT>/docker` directory. ::
+
+        cd ..
+
+Deployment profiles
+****************************
+
+Deployment profiles are defined to parameterize your deployment, either you only want to run the server side core components of the registry (OpenSearch and web API) or run an harvest standalone job, or run the scalable harvest services...
+
+Using a profile name, you can start the required components of the Registry as follows. ::
+
+    docker compose --profile=<DOCKER_COMPOSE_PROFILE_NAME> up --detach
+
+.. note::
+    The next section explains the possible profile names that can be used for the `<DOCKER_COMPOSE_PROFILE_NAME>` in the
+    docker compose command above.
+
+The docker containers deployed using a docker compose profile can be easily uninstalled and cleaned-up using the following
+command::
+
+    docker compose --profile=<DOCKER_COMPOSE_PROFILE_NAME> down
+
+
+The following table contains commonly used server-side profiles and descriptions.
+
+====================== ==================================================== ==============================================
+ Profile Name           Description                                          Prerequisites
+
+                                                                             (profiles to start before this)
+
+====================== ==================================================== ==============================================
+ pds-core-registry      Starts only the OpenSearch and Registry API          None
+ pds-loader-services    Starts the Scalable Harvest server-side components   The `pds-core-registry` profile
+                                                                             must be up and running
+====================== ==================================================== ==============================================
+
+
+Client-side operations
```

Comment:
Title Case Suggestion:  **Client-side Operations**

It is better to use  the  [Title Case ](https://en.wikipedia.org/wiki/Title_case) in this heading to be consistent with the Title Cases used on most of the headings in this page.

Reply:
Done